sdk/spring/spring-messaging-azure-eventhubs/src/main/java/com/azure/spring/messaging/eventhubs/core/properties/CommonProperties.java

```diff
@@ -11,7 +11,7 @@
 /**
  * Common properties shared by event hub namespace, a producer, and a consumer.
  */
-abstract class CommonProperties extends AzureAmqpSdkProperties implements EventHubClientCommonProperties, ConnectionStringProvider {
+public abstract class CommonProperties extends AzureAmqpSdkProperties implements EventHubClientCommonProperties, ConnectionStringProvider {
```

Comment:
why do we need to change this?

Reply:
Because the class com.azure.spring.cloud.nativex.configuration.implementation.messaging.MessageEventHubsHints refers it.